distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -199,17 +224,17 @@ Run the ${branding} using the appropriate script.
 
 .*NIX
 ----
-<${branding}_HOME/bin/${branding-lowercase}
+<${branding}_HOME/bin/start
```

Comment:
Don't think we should change this here. If we do keep this then we need to add instructions to run the client script afterward.